API/src/main/java/net/juligames/core/api/config/ConfigurationAPI.java

```diff
@@ -29,5 +32,65 @@ public interface ConfigurationAPI {
 
     Comparator<? extends Configuration> comparator();
 
-    //buildInInterpreters
+    //collections on demand - 1.1
+
+    /**
+     * This will split the given collection using the given interpreter
+     *
+     * @param collection  the collection
+     * @param interpreter the interpreter
+     * @param <T>         Type
+     * @return a {@link Collection} out of the interpreted Ts
+     */
+    <T> Collection<String> split(Collection<T> collection, Interpreter<T> interpreter);
```

Comment:
Confusing method name
```suggestion
    <T> Collection<String> map(Collection<T> collection, Interpreter<T> interpreter);
```

Reply:
In the case, you want to change this to map, then it would require a huge refactor over a lot of different files to avoid confusing method names in the Configurations

Reply:
ok